src/content/en/updates/2017/02/media-session.md

```diff
@@ -0,0 +1,445 @@
+
+#### Previous Track / Next Track
+
+If your web app provides a playlist for instance, you may want to allow the
+user to navigate through your playlist directly from the media notification
+with some "Previous Track" and "Next Track" icons.
+
+    let audio = document.createElement('audio');
+
+    let playlist = ['audio1.mp3', 'audio2.mp3', 'audio3.mp3'];
+    let index = 0;
+    
+    navigator.mediaSession.setActionHandler('previoustrack', function() {
+      // User clicked "Previous Track" media notification icon.
+      index = (index - 1 + playlist.length) % playlist.length;
+      playAudio();
+    });
+    
+    navigator.mediaSession.setActionHandler('nexttrack', function() {
+      // User clicked "Next Track" media notification icon.
+      index = (index + 1) % playlist.length;
+      playAudio();
+    });
+
+    function playAudio() {
+      audio.src = playlist[index];
+      audio.play()
+      .then(_ => { /* Set up Media Session... */ })
+      .catch(error => { console.log(error); });
+    }
+
+    playButton.addEventListener('pointerup', function(event) {
+      playAudio();
+    });
+
+
+Note that media action handlers will persist. This is very similar to the event
+listener pattern except that handling an event means that the browser steps
+doing any default behaviour and uses this as a signal that your web app
+supports the media action. Hence, media action controls won't be shown unless
+you set the proper action handler.
+
+By the way, unsetting a media action handler is as easy as assigning it to `null`.
+
+#### Seek Backward / Seek Forward
+
+The Media Session API allows you to show "Seek Backward" and "Seek Forward"
+media notification icons if you want to control the amount of time skipped.
+
+    let skipTime = 10; // Time to skip in seconds
+
+    navigator.mediaSession.setActionHandler('seekbackward', function() {
+      // User clicked "Seek Backward" media notification icon.
+      audio.currentTime = Math.max(audio.currentTime - skipTime, 0);
+    });
+    
+    navigator.mediaSession.setActionHandler('seekforward', function() {
+      // User clicked "Seek Forward" media notification icon.
+      audio.currentTime = Math.min(audio.currentTime + skipTime, audio.duration);
+    });
+
+#### Play / Pause
+
+The "Play/Pause" icon is always shown in the media notification and the related
+events are handled automatically by the browser. If for some reason the default
+behaviour doesn't work out, you can still handle "Play" and "Pause" media events.
+
+    navigator.mediaSession.setActionHandler('play', function() {
+      // User clicked "Play" media notification icon.
+      // Do something more than just playing current audio...
+    });
+
+    navigator.mediaSession.setActionHandler('pause', function() {
+      // User clicked "Pause" media notification icon.
+      // Do something more than just pausing current audio...
+    });
+
+Note: As the browser may consider the web app is not be playing when media
```

Comment:
'The browser may consider that the web app is not playing media when files are seeking or loading. You can override ...'